grpclb_remote_balancer.go

```diff
@@ -176,13 +176,13 @@ func (lb *lbBalancer) sendLoadReport(s *balanceLoadClientStream, interval time.D
 	}
 }
 
-func (lb *lbBalancer) callRemoteBalancer() error {
+func (lb *lbBalancer) callRemoteBalancer() (backoff bool, _ error) {
```

Comment:
Another idea, to avoid the need for this boolean:

Remove `lb.readServerList()` from this function and call it where this function is called from.  Then do:

```
if err := callRemoteBalancer(); err != nil {
  // Do not backoff
  continue
}
if err := lb.readServerList(); err != nil {
  // Backoff
  ...
}
```

Reply:
Then the function still needs to return the stream as well.
Will keep the current code.